src/components/ProgressBar/index.spec.js

```diff
@@ -0,0 +1,58 @@
+import { describe, expect, it } from 'vitest';
+import ProgressBar from './index';
+
+describe('ProgressBar', () => {
+  it('should be a Function', () => {
```

Comment:
Vamos evitar o uso de should, afinal ele fica repetitivo e reduntante `it should...`

```suggestion
  it('will be a Function', () => {
```

Reply:
`it('is a function')`